web/styles/right_sidebar.css

```diff
@@ -269,6 +269,22 @@ $user_status_emoji_width: 24px;
         display: flex;
     }
 
+    .user-name-and-status-emoji.with-avatar {
+        display: flex;
+        height: 39px;
+        /* without this width, emojis can be partially cut off */
+        /* this is the maximum width that prevents this */
+        width: 168px;
```

Comment:
This should be computed using `calc` or some other method, or it'll break if we toggle the font size setting or any other adjacent CSS affecting widths changes. Or perhaps a better layout method should be used like flex/grid that doesn't require weird magic numbers.